browser_history/generic.py

```diff
@@ -377,3 +423,10 @@ def default(self, o):
             )
 
         return json_string
+
+    format_map: Dict[str, Any] = {
```

Comment:
We could have a [Callable](https://docs.python.org/3/library/typing.html#callable) instead of `Any`